.eslintignore

```diff
@@ -1,26 +1,26 @@
 # unconventional js
-packages/*/blueprints/*/*files/
-vendor/
+**/blueprints/*/*files/
+**/vendor/
 
 # compiled output
+**/dist/
+**/tmp/
 /packages/-ember-data/docs/
-dist
-tmp
 
 # dependencies
-bower_components
-node_modules
+**/bower_components/
+**/node_modules/
```

Comment:
this would not allow for ignore root level

Reply:
Are you sure about that? AFAICT the `.eslintignore` file follows the same semantics as `.gitignore`. This means to ignore any directories named `node_modules` anywhere in the repo, including the root.

From https://git-scm.com/docs/gitignore#_pattern_format:

> A leading `"**"` followed by a slash means match in all directories. For example, `"**/foo"` matches file or directory `"foo"` anywhere, the same as pattern `"foo"`. `"**/foo/bar"` matches file or directory `"bar"` anywhere that is directly under directory `"foo"`.

You can verify this if you make the same changes to `.gitignore` and either check to see that the root `node_modules` stays ignored or by running [`git check-ignore *`](https://git-scm.com/docs/git-check-ignore).

Reply:
Yeah reading glob rules seems I am mistaken here, although I swear this has not worked for me before :/